engines/griffon/cutscenes.cpp

```diff
@@ -182,6 +186,17 @@ void GriffonEngine::showLogos() {
 }
 
 void GriffonEngine::intro() {
+#ifdef USE_TTS
+	Common::TextToSpeechManager *_ttsMan = g_system->getTextToSpeechManager();
+	if (_ttsMan != nullptr) {
+		Common::String result = story[0];
+		for(int i=10; i< ARRAYSIZE(story); ++i){
```

Comment:
`for (int i = 10; i < ARRAYSIZE(story); ++i) {`

This would e..g. be the right formatting. Watch out for the missing whitespaces. See the existing code to get an idea about the formatting. Keep formatting it the same way.

Or try to use something like clang-formatter to perform the base formatting (if you ever have more than this to format)

Also if you use clang-format on a whole file, you should get used to `git add -p` to only use the relevant parts for your particular commit.